src/amalgam/gen/neondot.c

```diff
@@ -5958,3 +5962,160 @@ void xnn_qu8_igemm_minmax_rndnu_ukernel_4x8c4__neondot(
     }
   } while (nc != 0);
 }
+
+void kai_matmul_clamp_f32_qai8dxp1x8_qsu4cxp8x8_1x8x32_neon_dotprod(
```

Comment:
is there a template/source for this?  We dont manually put functions in amalgams.  Each kernel has its own source.